### PR TITLE
AppShell + Icons from Figma

### PR DESCRIPTION
**Pull request by hamstu**
image

* Decided to build out an `AppShell` component that we can all wrap our apps in. It's not quite done yet (hence _WIP_), but the idea is that it will be made up of 1) the NavBar, 2) a sidebar, and 3) the main content. You'll be able to pass in custom props/components to render custom things in the menus, sidebar, and of course the main content. 
* This will be built with mostly Publish in mind to start, but it will be something we use in Reply and Analyze as well with hopefully only a few tweaks.

## Icons

This PR also includes auto icon generation from Figma! See the `CONTRIBUTING.md` file for details.

### CODE REVIEW

File: src/documentation/app/AppContainer.jsx [hunk 1]
```diff
@@ -75,7 +54,7 @@ export default class AppContainer extends React.Component {
   renderMarkdownComponent = () => <markdownToJsx>{UIComponent}</markdownToJsx>
 
   render() {
-    const { route, location } = this.state;
+    const { match: { params: { route, location, view } } } = this.props;
```
Comment (ivanaszuber) [inline src/documentation/app/AppContainer.jsx]: Ah, thanks so much for fixing this @hamstu 🙇 !

File: src/documentation/app/layout/content/Component.jsx [hunk 2]
```diff
@@ -49,26 +49,39 @@ const ExampleWrapper = style.div`
...
-      // if this component example contains subfolders, then get example from each subfolder
-      ? folder.map((example, idx) => <Example key={name + idx} example={example} componentName={name} id={id} />)
-      // otherwise just render the example
-      : <Example example={folder} componentName={name} id={id} key={name} />
-    }
-  </ExampleWrapper>,
-];
+  fullscreen, name, folder, id,
+}) => (
+  <Fragment>
```
Comment (ivanaszuber) [inline src/documentation/app/layout/content/Component.jsx]: Ah, this is great! I'm always defaulting to using the array `[ ]`, but the `fragments` and `<>` feels like a better approach 👌

File: src/documentation/app/AppContainer.jsx (hunk 1 above)
Comment (hamstu) [reply src/documentation/app/AppContainer.jsx]: Just a little bit of healthy destructuring! 😂

File: src/documentation/app/layout/content/Component.jsx (hunk 2 above)
Comment (hamstu) [reply src/documentation/app/layout/content/Component.jsx]: Yeah totally! Love this feature of React. 😍

**Pull request merged**